Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -156,6 +167,10 @@ public void RegisterPointer(IPointingSource pointingSource)
                 {
                     gazeManagerPointingData = new PointerData(pointingSource);
                 }
+                else
+                {
+                    gazeManagerPointingData.ResetFocusedObjects();
```

Comment:
This fixes a bug.  When switching between a 6DoF controller's pointing ray and gaze, if the user is staring at an object, the focus wasn't updated correctly.  This change fixes that case.